<launch>
	<node pkg="CamTrap_Viper" type="node_lowres.py" name="node_lowres"/>		
</launch>
